frequency: 20
frame_id: "gyro_link"

# Hardware parameters
i2c_bus: 1
device_addr: 0x68
